### ISSUE TITLE
How to save the fine-tuned model

### ISSUE THREAD

**Issue by yz-qiang** [huggingface/transformers#14974]
Hi, 
I save the fine-tuned model with the `tokenizer.save_pretrained(my_dir)` and `model.save_pretrained(my_dir)`. Meanwhile, the model performed well during the fine-tuning(i.e., the loss remained stable at **0.2790**). And then, I use the `model_name.from_pretrained(my_dir)` and ` tokenizer_name.from_pretrained(my_dir)` to load my fine-tunned model, and test it  in the training data. The value of loss surprised me because it was so high, i.e., **4.7**. I don't know why. Can you help me?  Please reply to me at your convenience. Thank you very much. :)
